.github/workflows/transifex_push_pull.yml

```
@@ -0,0 +1,93 @@
+name: "Transifex Push / Pull"
+
+on:
+  workflow_call:
+    secrets:
+      tx_token:
+        required: true
+      based_branch:
+        required: true
+
+jobs:
+  ci:
+    name: "Translations"
+    runs-on: ubuntu-latest
+    steps:
+
+    ##########################################
+    # Checkout repository                    #
+    ##########################################
+    - name: "Checkout"
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ secrets.based_branch }}
+        fetch-depth: 0 #To Fetch All Tags and Branches
+
+    ##########################################
+    # Install depdendencies                  #
+    ##########################################
+    - name: Install Dependencies
+      uses: php-actions/composer@v6
+      with:
+        dev: yes
+
+    - name: Install xgettext
+      run: sudo apt-get install gettext libgettextpo-dev
+
+    ##########################################
+    # Generate <plugin>.pot file             #
+    ##########################################
+    - name: "Generate <plugin>.pot file"
+      run: ./vendor/bin/extract-locales
+
+
+    ##########################################
+    # Push and Pull transifiex source        #
+    ##########################################
+    - name: Push source file (.pot)
+      uses: transifex/cli-action@v2
+      with:
+        args: push
+        token: ${{ secrets.tx_token }}
+
+    - name: Clean Transifex CLI
+      run: rm -fr /tmp/tx
+
+    - name: Pull source file (.po / .mo)
+      uses: transifex/cli-action@v2
+      with:
+        args: pull --force --all --minimum-perc=80
+        token: ${{ secrets.tx_token }}
+
+    ##########################################
+    # Generate .mo files                     #
+    ##########################################
+    - name: "Compil mo"
+      run: ./vendor/bin/plugin-release --compile-mo
+
+    ##########################################
+    # Generate random branch name            #
+    ##########################################
+    - name: generate FOO unique variable based on timestamp
+      run: echo BRANCH_NAME=transifex/i18n-updates--$(date +%s) >> $GITHUB_ENV
+
+
+    ##########################################
+    # Create Pull Request                    #
+    ##########################################
+    - name: Create Pull Request
```

I think it will create a new PR everytime because it will, at least, change the generation date inside the pot and the en_GB.po files.
A new PR should be created only if the `mo` files are updated (it means that something changed on transifex side, either a new translation, either a change in translation authors). This could be done by something like this
```
- name: "Check status"
  id: "check-status"
  run: |
    if [[ `git status --porcelain locales/*.mo` ]]; then
      echo "create_pr=yes" >> $GITHUB_OUTPUT
    else
      echo "create_pr=no" >> $GITHUB_OUTPUT
    fi
```
and this condition:
```suggestion
    - name: Create Pull Request
      if: ${{ steps.check-status.outputs.create_pr == 'yes' }}
```

---

agree with that